### PR TITLE
AccessMana is now Mana1

### PR DESCRIPTION
**Pull request by karimodm**
* package/mana/access* all gone
* consensusbase renamed to manabase implementing base interface for both access and consensus mana
* consensusbasevector renamed to manabasevector implementing basevector interface for both access and consensus mana
* both mana vectors are now maintained using 2 instances of ManaBaseVector (former ConsensusBaseVector)
* update methods removed from both interfaces and implementations
* SetCoefficients gone, along with corresponding mana plugin parameters
* EffectiveValue methods removed, only BaseValue remains
* manarefresher is gone
    * DelegationAddress stuff managed by the plugin also gone
        * Reclaim functionality removed
    * corresponding WebAPI endpoints and response fields removed
    * cli-wallet delegation & reclaim removed
    * occurrences removed from Ansible and docker-compose files
* GetMana and GetManaMap methods do not receive and optional time anymore, but they still return a timestamp which is always set to time.Now(). This timestamp should be tied to the epoch timestamp for the committed mana vector.

### CODE REVIEW

File: packages/mana/basevector.go [hunk 1]
```diff
@@ -48,9 +44,9 @@ type BaseManaVector interface {
 func NewBaseManaVector(vectorType Type) (BaseManaVector, error) {
```
Comment (piotrm50) [inline packages/mana/basevector.go:44]: Do we need this function now? Both cases in the switch perform the same action anyway

File: packages/mana/events_test.go [hunk 2]
```diff
@@ -92,8 +92,8 @@ func newRevokeEvent() *RevokedEvent {
 func newUpdateEvent() *UpdatedEvent {
 	return &UpdatedEvent{
 		NodeID:   randomNodeID(),
-		OldMana:  &ConsensusBaseMana{},
-		NewMana:  &ConsensusBaseMana{},
+		OldMana:  &ManaBase{},
+		NewMana:  &ManaBase{},
```
Comment (piotrm50) [inline packages/mana/events_test.go:96]: This is safer as it already initializes the models.

```suggestion
		OldMana:  NewManaBase(0.0),
		NewMana:  NewManaBase(0.0),
```

File: plugins/messagelayer/parameters.go [hunk 3]
```diff
@@ -29,12 +29,6 @@ type ParametersDefinition struct {
 
 // ManaParametersDefinition contains the definition of the parameters used by the mana plugin.
 type ManaParametersDefinition struct {
-	// EmaCoefficient1 defines the coefficient used for Effective Base Mana 1 (moving average) calculation.
```
Comment (piotrm50) [inline plugins/messagelayer/parameters.go:32]: Did we want to remove the coefficient for mana1 as well? Or was it never actually used?

File: plugins/metrics/mana.go [hunk 4]
```diff
@@ -202,6 +201,4 @@ func measureMana() {
 		consensusAvg = consensusSum / float64(len(neighborConsensusMap))
 	}
 	averageNeighborsConsensus.Store(consensusAvg)
-
-	delegationAmount.Store(manarefresher.TotalDelegatedFunds())
```
Comment (piotrm50) [inline plugins/metrics/mana.go:206]: This is the only place where we set value of this variable. We should remove it completely from the metrics.

**Review by piotrm50 — CHANGES_REQUESTED** [review]
Added comments for some minor changes.

File: plugins/messagelayer/parameters.go (hunk 3 above)
Comment (karimodm) [reply plugins/messagelayer/parameters.go:32]: It was never actually used!

File: packages/mana/basevector.go (hunk 1 above)
Comment (karimodm) [reply packages/mana/basevector.go:44]: True! I've turned it into a basic constructor.

**Review by jonastheis — APPROVED** [review]
LGTM but I think you'll need to adjust the (mana) integration tests as well

**Comment by karimodm** [comment]
Yeah... will fix all the tests in the base branch.

**Comment by karimodm** [comment]
Yeah... will fix all the tests in the base branch.

**Review by piotrm50 — APPROVED** [review]

**Pull request merged**